.gitignore

```diff
@@ -2,3 +2,4 @@ build
 profiling
 benchmarks
 dump.txt
+.DS_Store
```

Comment:
```suggestion
**/.DS_Store
```

Reply:
Looks like works without it, but ok :)

Reply:
ok. was not sure of the default behaviour of .gitignore here